plugin.js

```diff
@@ -36,12 +36,16 @@ function fastifyCookieClearCookie (reply, name, options) {
   return fastifyCookieSetCookie(reply, name, '', opts)
 }
 
+function fastifyParseCookie (cookieHeader, options) {
+  return cookie.parse(cookieHeader, options)
+}
+
 function onReqHandlerWrapper (options) {
   return function fastifyCookieOnReqHandler (fastifyReq, fastifyRes, done) {
     fastifyReq.cookies = {} // New container per request. Issue #53
     const cookieHeader = fastifyReq.raw.headers.cookie
     if (cookieHeader) {
-      fastifyReq.cookies = cookie.parse(cookieHeader, options)
+      fastifyReq.cookies = fastifyParseCookie(cookieHeader, options)
```

Comment:
please use the decorator function instead

Reply:
Do I have to pass the fastify instance to the onReqHandlerWrapper or is there a better way of doing this?